cockroachdb-parent/charts/cockroachdb/templates/crdb.yaml

```
@@ -128,3 +128,7 @@ spec:
       {{- with .Values.cockroachdb.crdbCluster.persistentVolumeClaimRetentionPolicy }}
       persistentVolumeClaimRetentionPolicy: {{- toYaml . | nindent 8 }}
       {{- end }}
+      {{- if .Values.cockroachdb.crdbCluster.virtualCluster }}
+      virtualCluster:
+        mode: {{ .Values.cockroachdb.crdbCluster.virtualCluster.mode }}
+      {{- end }}
```

nit: would the operator default the mode to `disabled`, if the `virtualCluster` section is empty?

```suggestion
      {{- with .Values.cockroachdb.crdbCluster.virtualCluster }}
      virtualCluster: {{- toYaml . | nindent 8 }}
      {{- end }}
```

---

Yes. Virtual cluster spec will be ommitted in crdbcluster spec. It also means we don't pass any flag (--virtualized or --virtualized-empty) to cockroach init command.